onnxruntime/core/providers/ngraph/ngraph_execution_provider.cc

```diff
@@ -320,7 +301,7 @@ static std::map<std::string, std::set<std::string>> GetNgSupportedOps(const int
   std::map<std::string, std::set<std::string>> ng_supported_ops;
   ng_supported_ops.emplace(kOnnxDomain, ngraph::onnx_import::get_supported_operators(onnx_opset, kOnnxDomain));
 
-  const std::set<std::string> ng_disabled_ops = {"LSTM", "Gather"};  //Place-holder for ops not supported.
+  const std::set<std::string> ng_disabled_ops = {"LSTM"};  //Place-holder for ops not supported.
```

Comment:
for me, GatherOpTest.Gather_axis2 and 
GatherOpTest.Gather_axis1_neg_indices2d_int8 fail
probably related to the issue brought up that nGraph's gather doesn't yet support negative indices.

Reply:
We've got a fix for the failing tests. It will be added to the nGraph 0.26 branch and the final tag that will be used in ONNXRT.

Reply:
we need to get the build succeeding first. are you testing with --build_wheel option when building? Our CI uses that flag.

Reply:
The Gather unit tests were failing because of a small bug in nGraph. This is what we have a fix for.

The issue with the `--build_wheel` flag was about `libcodegen.so`, this is already fixed by the commit that forces MKLDNN 1.0 to be used.

Reply:
thanks. looks like there's a new test failure.
https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=82258